docs/explore_api/tasks.md

```diff
@@ -0,0 +1,210 @@
+# Tasks
+
+Central to the API's functionality is the concept of a `Task`, which is an 
+abstraction that encapsulates all the information about the
+computational workload you defined as the user. Once you submit a simulation to the 
+API, a `Task` is generated and remains there until you explicitly delete it. This 
+allows for real-time updates on simulation status, including monitoring 
+its progress, retrieving its outputs, and adjusting parameters or re-running simulations 
```

Comment:
The last to points are also not true at the moment. Not sure if we should keep them.